Libraries/Text/RCTTextFieldManager.m

```diff
@@ -78,6 +78,7 @@ - (BOOL)textFieldShouldEndEditing:(RCTTextField *)textField
 RCT_REMAP_VIEW_PROPERTY(editable, enabled, BOOL)
 RCT_EXPORT_VIEW_PROPERTY(placeholder, NSString)
 RCT_EXPORT_VIEW_PROPERTY(placeholderTextColor, UIColor)
+RCT_EXPORT_VIEW_PROPERTY(tintColor, UIColor)
```

Comment:
Can you expose this in RCTTextViewManager.m too please, as TextInput.js wraps them both.